routing/login.js

```diff
@@ -36,7 +36,7 @@ const compare = (req, res, user) => {
   const password = user.password;
   bcrypt.compare(password, hash, (err, result) => {
     if (err) {
-      console.log(err)
+      console.log(err);
```

Comment:
:smile:  doog job)

Reply:
Actually, I don't remember myself doing this) I think it happened while merging